Microsoft.Toolkit.Uwp.UI/Extensions/Mouse/Mouse.cs

```diff
@@ -0,0 +1,114 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+   /// <summary>
+   /// Helper class for easily changing the mouseover cursor type.
+   /// </summary>
+   public class Mouse : DependencyObject
+   {
+      /// <summary>
+      /// Dependency property for specifying the target <see cref="CoreCursorType"/> to be shown
+      /// over the target <see cref="FrameworkElement"/>.
+      /// </summary>
+      public static readonly DependencyProperty CursorProperty =
+          DependencyProperty.RegisterAttached("Cursor", typeof(CoreCursorType), typeof(Mouse), new PropertyMetadata(CoreCursorType.Arrow, CursorChanged));
+
+      /// <summary>
+      /// Set the target <see cref="CoreCursorType"/>.
+      /// </summary>
+      /// <param name="element">Object where the selector cursor type should be shown.</param>
+      /// <param name="value">Target cursor type value.</param>
+      public static void SetCursor(FrameworkElement element, CoreCursorType value)
+      {
+         element.SetValue(CursorProperty, value);
+      }
+
+      /// <summary>
+      /// Get the current <see cref="CoreCursorType"/>.
+      /// </summary>
+      /// <param name="element">Object where the selector cursor type should be shown.</param>
+      /// <returns>Cursor type set on target element.</returns>
+      public static CoreCursorType GetCursor(FrameworkElement element)
```

Comment:
Can we use DependencyObject instead of FrameworkElement

Reply:
DependencyObject doesn't have the Pointer and Unloaded events needed to drive the cursor changes though.  We'd still need to cast at some point, but if we opened to DO, then it'd show up in more places than it'd actually function.

Reply:
Same answer as above.